src/main/kotlin/apply/ui/admin/evaluation/EvaluationForm.kt

```diff
@@ -0,0 +1,67 @@
+package apply.ui.admin.evaluation
+
+import apply.application.EvaluationRequest
+import apply.domain.evaluation.Evaluation
+import apply.domain.recruitment.Recruitment
+import com.vaadin.flow.component.button.Button
+import com.vaadin.flow.component.textfield.TextArea
+import com.vaadin.flow.component.textfield.TextField
+import support.views.BindingFormLayout
+import support.views.createErrorSmallButton
+import support.views.createItemSelect
+import support.views.createPrimarySmallButton
+
+class EvaluationForm() : BindingFormLayout<EvaluationRequest>(EvaluationRequest::class) {
+    private val title: TextField = TextField("평가명")
+    private val description: TextArea = TextArea("설명")
+    private val recruitment = createItemSelect<Recruitment>("모집").apply {
```

Comment:
프로퍼티는 자료형을 한 눈에 알아 볼 수 있게 하면 어떨까요?

```suggestion
    private val recruitment: Select<Recruitment> = createItemSelect<Recruitment>("모집").apply {
```